api/pom.xml

```diff
@@ -362,6 +362,28 @@
 					<header>${project.parent.basedir}/license-header.txt</header>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<createDependencyReducedPom>false</createDependencyReducedPom>
+							<shadedArtifactAttached>true</shadedArtifactAttached>
+							<shadedClassifierName>shaded</shadedClassifierName>
```

Comment:
@ibacher, I think the API jar is still there and published as usual, the shaded jar is just an extra deliverable for the CI snapshot process